Orange/tests/test_clustering_kmeans.py

```diff
@@ -2,64 +2,134 @@
 # pylint: disable=missing-docstring
 
 import unittest
+import warnings
 
 import numpy as np
-from scipy.sparse import csc_matrix
+from scipy.sparse import csc_matrix, csr_matrix
```

Comment:
I find it a bit strange, that we already tested learning on csc, but you added csr for testing the models. Even worse is that giving a model a csc matrix would result in a crash (although this is avoided in tests)...
I know this is baggage inherited from the copy paste of call from learning models, but I think it needs fixing (probably in a separate PR).